### PR TITLE
Configure plugin for automated releases

### PR DESCRIPTION
**Pull request by dwnusbaum**
See https://github.com/jenkins-infra/repository-permissions-updater/pull/3484.

### Testing done

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
 
   <artifactId>display-url-api</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${revision}.${changelist}</version>
   <packaging>hpi</packaging>
 
   <properties>
-    <revision>2.3.9</revision>
+    <revision>2</revision>
     <changelist>-SNAPSHOT</changelist>
```
Comment (NotMyFault) [inline pom.xml:18]:
```suggestion
    <changelist>999999-SNAPSHOT</changelist>
```

Comment (dwnusbaum) [reply pom.xml:18]: Ah sorry, thanks

Comment (NotMyFault) [reply pom.xml]: Nw, thanks for applying!

**Comment by olamy** [comment]
I'm happy to keep releasing it..

**Comment by dwnusbaum** [comment]
@olamy yes, but with so many other plugins already using CD releases, why not use it here too? I can release manually as well (at least when my artifactory credentials are not broken), but automated releases are simpler. The fact that we use GitHub actions and not Jenkins for this purpose is awkward and unfortunate but it is what it is.

Do you want me to close this?

**Comment by olamy** [comment]
> @olamy yes, but with so many other plugins already using CD releases, why not use it here too? 

ah right majority in democracy. I love the quotes around this topic such

`The majority is always wrong; the minority is rarely right`
Henrik Ibsen

Or 

`Sometimes the majority just means all the fools are on the same side`
John Kennedy

You know my opinion about those "Powered by GitHub Actions" or "Delivered by GitHub Actions" 
Sadly I do not have enough photoshop skills to make a nice logo we could add in plenty of Jenkins repositories.

> I can release manually as well (at least when my artifactory credentials are not broken), but automated releases are simpler. The fact that we use GitHub actions and not Jenkins for this purpose is awkward and unfortunate but it is what it is.
> 
> Do you want me to close this?

Do whatever you want but do not ask me for approval ;) as said previously the majority in democracy wins so the majority just turns quiet but still has the freedom of speech :laughing:

**Review by car-roll — APPROVED** [review]

**Comment by olamy** [comment]
Really to ask GitHub actions sponsoring for proper marketing purpose

**Pull request merged**